projects/gfbio-app/src/app/app.component.ts

```diff
@@ -240,7 +240,10 @@ export class AppComponent implements OnInit, AfterViewInit {
                     this.dialog.open(BasketDialogComponent, {data: {result}});
                 });
             } else {
-                this.dialog.open(SplashDialogComponent, {});
+                const showSplash = this.userService.getSettingFromLocalStorage('showStartupSplashScreen');
```

Comment:
…here, so that you are sure to always have the correct key.

Reply:
[115f7bf](https://github.com/arianegr/geoengine-ui/commit/115f7bf06a2073675888913a3f4a1a5f89b0fdd2)